doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -43,9 +43,13 @@ commands that operate on pname:instance or a child of pname:instance can: be
 obtained. The returned function pointer must: only be called with a
 dispatchable object (the first parameter) that is a child of pname:instance.
 
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
+If pname:pName is not the name of one of these Vulkan commands, and is not
```

Comment:
Was there before and perhaps should be done to trivialy fix the immediate problem of supportted vs enabled. But IMHO it has these issues:

1) Redefines what is said above. For perfection would need a chapter rewrite so that all of that links to same definition of "core Vulkan command" and "extension command".

2) What about the case `instance == NULL` and getting core  command or extension command?

3) What about the case of `instance == valid` and getting freestanding command?